src/recode.jl

```diff
@@ -310,16 +310,16 @@ julia> using CategoricalArrays, Missings
 
 julia> recode(1:10, 1=>100, 2:4=>0, [5; 9:10]=>-1, 6=>missing)
 10-element Array{Union{Int64, Missings.Missing},1}:
- 100    
```

Comment:
I think the spaces are correct, they reflect the REPL output.

Reply:
my bad, I didn't see that those got committed by accident...